Add instructions for embedded images: (#9076)
aws/eks-anywhere: docs/content/en/docs/getting-started/baremetal/customize/bare-custom-hookos.md

    cd hook/
    ```

> 1. Run the Linux kernel [menuconfig](https://en.wikipedia.org/wiki/Menuconfig) TUI and configuring the kernel as needed. Be sure to save the config before exiting.

Reviewer: ```suggestion
1. Run the Linux kernel [menuconfig](https://en.wikipedia.org/wiki/Menuconfig) TUI and configure the kernel as needed. Save the config before you exit.
```

    ./build.sh kernel hook-latest-lts-amd64
    ```

> 1. Add the embedded Action images. This creates the file, `images.txt`, in the `images/hook-embedded` directory and runs the script, `images/hook-embedded/pull-images.sh`, to pull and embedded the images in the HookOS initramfs.

Reviewer: ```suggestion
1. Add the embedded Action images. This creates the file, `images.txt`, in the `images/hook-embedded` directory and runs the script, `images/hook-embedded/pull-images.sh`, to pull and embed the images in the HookOS initramfs.
```
